lib/ajax-datatables-rails/orm/active_record.rb

```diff
@@ -38,31 +38,28 @@ def composite_search(records)
         records
       end
 
-      def build_conditions_for(query)
+      def build_conditions_for(query, regex)
         search_for = query.split(' ')
         criteria = search_for.inject([]) do |criteria, atom|
-          criteria << searchable_columns.map { |col| search_condition(col, atom) }
+          criteria << searchable_columns.map { |col| search_condition(col, atom, regex == 'true') }
             .reduce(:or)
         end.reduce(:and)
         criteria
       end
 
       def aggregate_query
-        conditions = searchable_columns.each_with_index.map do |column, index|
+        conditions = view_columns.each_with_index.map do |column, index|
```

Comment:
thinking we actually need `searchable_columns` here, given not all columns will be marked as searchable when coming in from the `params` hash that the plugin provides.

You can see the current implementation of `searchable_columns` here: https://github.com/antillas21/ajax-datatables-rails/blob/v-0-4-0/lib/ajax-datatables-rails/base.rb#L42-L45